client/my-sites/email/inbox-management/mailbox-selection-list/progress-line-style.scss

```
@@ -0,0 +1,101 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.progress-line__container {
+	position: fixed;
+	top: 50%;
+	left: 50%;
+	transform: translate( -50%, -50% );
+
+	@include break-large {
+		transform: none;
+	}
+
+
+	.progress-line__status-text-container {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		margin-bottom: 10px;
+
+		@include break-large {
+			margin-bottom: 20px;
+		}
+
+		.progress-line__status-text {
+			font-family: SF Pro Display, SF UI Text, SFUIText-Regular, Helvetica Neue, HelveticaNeue, Helvetica, Arial, sans-serif;
```

Please use `$sans` instead as per [the Typography docs](https://wpcalypso.wordpress.com/devdocs/typography).
```suggestion
			font-family: $sans;
```

---

TIL